src/app/app.module.ts

```diff
@@ -20,8 +20,117 @@ export const DEFAULT_LIBRARY_BUTTONS = {
     styleClass: 'italic',
     icon: 'icon-italic'
   },
+  fontName: {
+    label: 'F',
+    title: 'Font name',
+    children: [
+      {
+        label: 'Arial',
+        title: 'Arial',
+        command: (ctx: NgxWigComponent) => {
+          ctx.execCommand('fontName', 'Arial')
+        },
+        styleClass: 'font-arial'
+      },
+      {
+        label: 'Courier',
+        title: 'Courier',
+        command: (ctx: NgxWigComponent) => {
+          ctx.execCommand('fontName', 'Courier')
+        },
+        styleClass: 'font-courier'
+      },
+      {
+        label: 'Times',
+        title: 'Times',
+        command: (ctx: NgxWigComponent) => {
+          ctx.execCommand('fontName', 'Times')
+        },
+        styleClass: 'font-times'
+      }
+    ]
+  },
+  specialChar: {
+    label: 'à',
+    title: 'Accented Characters',
+    children: [
+      {
+        label: 'à',
+        title: 'à',
+        command: (ctx: NgxWigComponent) => {
+          ctx.execCommand('insertText', 'à')
+        }
+      },
+      {
+        label: 'è',
+        title: 'è',
+        command: (ctx: NgxWigComponent) => {
+          ctx.execCommand('insertText', 'è')
+        }
+      },
+      {
+        label: 'é',
+        title: 'é',
+        command: (ctx: NgxWigComponent) => {
+          ctx.execCommand('insertText', 'é')
+        }
+      },
+      {
+        label: 'ì',
+        title: 'ì',
+        command: (ctx: NgxWigComponent) => {
+          ctx.execCommand('insertText', 'ì')
+        }
+      },
+      {
+        label: 'ò',
+        title: 'ò',
+        command: (ctx: NgxWigComponent) => {
+          ctx.execCommand('insertText', 'ò')
+        }
+      },
+      {
+        label: 'ù',
+        title: 'ù',
+        command: (ctx: NgxWigComponent) => {
+          ctx.execCommand('insertText', 'ù')
+        }
+      },
+    ],
+    visibleDropdown: false
+  },
+  removeFormatting: {
+    label: 'R',
+    title: 'Remove formatting',
+    command: 'removeFormat'
+  },
+  smallcaps: {
+    label: 'S',
+    title: 'Small-caps',
+    command: (ctx: NgxWigComponent) => {
+      setSmallcaps(ctx);
+    }
+  }
 };
 
+function setSmallcaps(ctx: NgxWigComponent) {
+  console.log(ctx);
+  let selection = window?.getSelection()?.getRangeAt(0).cloneContents();
```

Comment:
I think you can make "window?.getSelection()?.getRangeAt(0)" shortcut as well to make it more readable